### ISSUE TITLE
High idle CPU usage

### ISSUE THREAD

**Issue by tmo1** [jliljebl/flowblade-forum#75]
Hi,

I'm new to Flowblade (2.4.0 on Debian unstable, from the distro package), and I'm puzzled by its high idle CPU usage. I'm running on a W550s with an i7-5500U @2.4 GHz, and while I understand that this is not the ideal video editing machine, I don't understand why the program uses so much CPU while doing nothing. I simply run 'flowblade' without opening any project or importing any files, and Flowblade uses about 45% of a CPU core, and remains at that usage level indefinitely. Is this normal?

**Comment by jliljebl** [comment]
Yeah, this was on the *wontfix* list for a long time, but this will be changed at some point in the future.

Currently Flowblade keeps producing frames at Project fps even when playback is paused, but now that we have async rendering for some project resources we are losing some performance here not just additional power usage.

**Comment by tmo1** [comment]
Okay, thanks. I've currently given up on Flowblade since I realized that since all I really want to do is trim and concatenate video clips, I really want a solution that doesn't involve rendering the output, so I've moved to a simple ffmpeg cli based workflow, but I'll give Flowblade another try if I ever need to do any real editing.

**Comment by lillemets** [comment]
Is there any progress on this issue? Just waiting Flowblade to be done producing frames seems futile. Having fans run at full speed even on idle is very annoying. Flowblade is otherwise great but I can't currently justify this waste of electricity.

**Comment by bmbeverst** [comment]
Appreciate the explanation. Also noticed this issue but at 60-70% of the CPU on KDE Neon.

**Comment by narical** [comment]
Ok, we're in 2025. I add ONE clip, make ONE filter - "Position Scale Rotate", add 4 keyframes.
From that point, program is unusable with constant 85-90% CPU load.
"Unusable" means constant slowdowns, mouse lagging etc.
Maybe I'm doing something wrong

Profile is FHD 1920x1080 30fps
CPU definitely isn't a moster, Intel Core i3-10100, 8 cores
GPU is RTX2080 but I guess that's not used
RAM 32 Gb

**Comment by jliljebl** [comment]
The *High idle CPU* issue described here has nothing to do with performance issue you are experincing.

Filter "Position Scale Rotate" is already optimized to take advantage of multiple cores by MLT, but the performance is what is now. And frankly yes, it is not very good. 

Performance of your system is at a level that it can can only be made be suitable for video editing when utilizing GPU for applying filters, and the only development solution for Flowblade is to move to do that as much as possible. 

There is no ifs, buts or what-ifs about this, modern video editing solutions should absolutely run end-to-end on GPU.

Unfortunately we are not there yet.

We will be implementing a "Create Rendered Filters Clip" stopgap solution which will allow to create clips from timeline clips with filters baked in, and that way get improved timeline playback on low performance systems 

You also already use proxies to drop encoding CPU usage to 12.5 - 25% of current levels and that will help here too.

**Comment by vacaboja** [comment]
I am in a similar situation to @narical above. Laptop, intel core i5-1130G7 with integrated graphics. Debian Trixie. Flowblade 2.20 (shipped by debian).

After opening a very simple project (two clips, one filter), my CPU usage remains pinned at about 80%. That happens **when flowblade is not playing** the preview and **not rendering.** Is this expected behavior, or do I have a problem in my setup?
